src/CISDSCResourceGeneration/functions/private/Import-GptTmpl.ps1

```diff
@@ -36,17 +33,15 @@ function Import-GptTmpl {
                         'Unicode' {Write-Verbose -Message 'Skipping Unicode section'}
                         #Version statement for the file that we also don't care about.
                         'Version' {Write-Verbose -Message 'Skipping Version section'}
-                        'Registry Values' {$ScaffoldingBlocks += ConvertFrom-RegistryValueRawGPO -Key $subkey -ValueData $ini[$key][$subKey]}
-                        'Privilege Rights' {$ScaffoldingBlocks += ConvertFrom-PrivilegeRightRawGPO -Policy $subkey -Identity $ini[$key][$subKey]}
-                        'System Access' {$ScaffoldingBlocks += ConvertFrom-SystemAccessRawGPO -Key $subkey -SecurityData $ini[$key][$subKey]}
-                        'Service General Setting' {$ScaffoldingBlocks += ConvertFrom-ServiceRawGPO -Service $subkey -ServiceData $ini[$key][$subKey]}
+                        'Registry Values' {ConvertFrom-RegistryValueRawGPO -Key $subkey -ValueData $ini[$key][$subKey]}
```

Comment:
Same concern with implicit return as above. 

If there's a good reason, by all means go for it - just suggesting a pause for consideration here.